task_sdk/tests/execution_time/test_task_runner.py

```diff
@@ -746,6 +746,13 @@ def test_get_connection_from_context(self, mocked_parse, make_ti_context, mock_s
             extra='{"extra_key": "extra_value"}',
         )
 
+        # Try and access the extra_dejson for conn
+        dejson_from_conn = conn_from_context.extra_dejson
+        assert dejson_from_conn == {"extra_key": "extra_value"}
+
+        # Try and access inner keys of dejson
+        assert conn_from_context.extra_dejson["extra_key"] == "extra_value"
```

Comment:
This is redundant since we assert a dict on L752

Reply:
Yeah, let me remove it